contracts/WildcardSteward_v3.sol

```diff
@@ -313,13 +316,16 @@ Y
                 collection
             );
 
-            // Collect the due loyalty tokens for the user
-            if (currentOwner != address(0)) {
-                _collectLoyaltyPatron(
-                    currentOwner,
-                    now.sub(deprecated_timeLastCollected[tokenId])
-                );
-            }
+            // mint required loyalty tokens
+            mintManager.tokenMint(
+                currentOwner,
+                timeSinceTokenLastCollection, // this should always be > 0
+                11574074074074 // instead of this -> tokenGenerationRate[tokenId] hard code to save gas
+            );
+            emit CollectLoyalty(
+                currentOwner,
+                timeSinceTokenLastCollection.mul(11574074074074)
+            ); // OPTIMIZE ME
```

Comment:
I almost wonder if we can leave this event out. It just uses gas, and doesn't really give much benefit IMO. In the upgrade event handler I will be able to check and update everyone's balance.